src/sites/signalen/components/signalen.css

```diff
@@ -0,0 +1,15 @@
+.signalen-form-navigation {
+  align-items: center;
+  display: flex;
+  gap: 1rem;
+  justify-content: space-between;
+}
+
+.signalen-summary-heading {
+  display: flex;
+  justify-content: space-between;
+}
+
+.amsterdam-link.amsterdam-link svg {
```

Comment:
Please use a `.signalen-back-link` class to prevent all link icons in all prototypes from flipping 😅